staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -228,3 +228,43 @@ func ipsToStrings(ips []net.IP) []string {
 	}
 	return ss
 }
+
+// ValidateCertificate checks that the given PEM-encoded data contains at least one
+// PEM block of type "CERTIFICATE", and that all blocks are of that type.
+func ValidateCertificate(pemData []byte) error {
```

Comment:
IMO, this function is better suited to be put in:
https://github.com/kubernetes/kubernetes/blob/71a5b255cc59512cec3993b7c29c86362ced7dc4/staging/src/k8s.io/client-go/util/cert/pem.go

and be called `ValidateCertificatePEM()`
or maybe it can be moved in another location, happy to hear alternative ideas, because certs.go doesn't feel like the right place.

ideally it should be somewhere with other Validate*/ cert functions